### PR TITLE
fix(324): Show error when no worker support is found

### PR DESCRIPTION
**Pull request by salmanm**
Fixes https://github.com/mcollina/autocannon/issues/324

### CODE REVIEW

File: test/run.test.js [hunk 1]
```diff
@@ -366,7 +366,7 @@ test('run should accept a unix socket/windows pipe', (t) => {
 })
 
 for (let i = 1; i <= 5; i++) {
-  test(`run should count all ${i}xx status codes`, (t) => {
+  test(`run should count all ${i}xx status codes`, { skip: true }, (t) => {
```
Comment (salmanm) [inline test/run.test.js:369]: Has to disable this temporarily as it fails consistently only on node 15.6.0.
We may need to investigate this separately.

Comment (mcollina) [reply test/run.test.js:369]: Can you please make it skip only on node v15?

**Review by mcollina — CHANGES_REQUESTED** [review]

File: test/run.test.js (hunk 1 above)
Comment (salmanm) [reply test/run.test.js]: Done

File: test/run.test.js [hunk 2]
```diff
@@ -366,7 +368,7 @@ test('run should accept a unix socket/windows pipe', (t) => {
 })
 
 for (let i = 1; i <= 5; i++) {
-  test(`run should count all ${i}xx status codes`, (t) => {
+  test(`run should count all ${i}xx status codes`, { skip: isNode15 }, (t) => {
```
Comment (mcollina) [inline test/run.test.js:371]:
```suggestion
  // TODO we should not skip those tests
  test(`run should count all ${i}xx status codes`, { skip: isNode15 }, (t) => {
```

**Review by mcollina — COMMENTED** [review]
tiny nit and we can land

File: test/run.test.js (hunk 2 above)
Comment (salmanm) [reply test/run.test.js:372]: Sure, I'm actually gonna look into the issue right after 🙂

**Review by mcollina — APPROVED** [review]

**Pull request merged**